Documentation/guides/building-apps/build-properties.md

```diff
@@ -549,6 +549,50 @@ The most common values for this property are:
 
 Added in Xamarin.Android 6.1.
 
+## AndroidIncludeWrapSh
+
+A boolean value which indicates whether the Android wrapper script
+([`wrap.sh`](https://developer.android.com/ndk/guides/wrap-script))
+should be packaged into the APK. The property defaults to `false`
+since the wrapper script may significantly influence the way the
+application starts up and works and the script should be included 
+only when necessary e.g. for debugging or otherwise changing the
+application startup/runtime behavior.
+
+The script is added to the project using the
+[`@(AndroidNativeLibrary)`](./build-items.md#androidnativelibrary)
```

Comment:
I fear this needs to be a "fuller" url, e.g. `(~/android/deploy-test/building-apps/build-items.md#androidnativelibrary)`, lest it cause complications when we next sync with MicrosoftDocs.

Reply:
This does *not* need to be a full URL; relative URLs are fine. https://teams.microsoft.com/l/message/19:446e3cd63c894194826e96b9d1b3032c@thread.skype/1602579498335?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=42121604-a67e-4f9a-9f3f-57c3c6d7ccd3&parentMessageId=1602541101300&teamName=Xamarin%20Platform%20Engineering&channelName=Docs&createdTime=1602579498335

Reply:
I changed it to use the same path as the other references in the document